scripts/main/init.js

```diff
@@ -95,6 +97,7 @@ $(document).ready(function () {
 				}
 			}
 		})
+		.bind(["h"], album.toggle_nsfw_filter)
```

Comment:
One more thing: this needs a `return false;` to prevent the keystroke from being interpreted by the browser itself.